storyblok/field-type-examples
# docs(StarRating): update star-rating docs to follow other example documentation

Issue: EXT-1902

## What?
It updates the docs of the Star Rating field plugin.

## Why?
To help users to understand better how to run the project locally and set it up correctly in their own spaces.

## How to test? (optional)
You can test in the sandbox/field plugin editor.
You can build the app and see if it works in the Visual Editor.

[[View Rendered →](https://github.com/storyblok/field-type-examples/blob/EXT-1902-update-docs/star-rating/README.md)]
